Dockerfile

```diff
@@ -1,13 +1,22 @@
 FROM node:12.18.4
 
-RUN mkdir /node-workspace
-COPY package.json /node-workspace 
-COPY package-lock.json /node-workspace
+# Default environment variables
+ENV ENV=dev USASPENDING_API=https://api.usaspending.gov/api/ MAPBOX_TOKEN='' GA_TRACKING_ID=''
```

Comment:
Will this overwrite the variables set [in the deploy script](https://github.com/fedspendingtransparency/usaspending-config/blob/master/ci/usaspending-frontend-prod.groovy#L82)? I believe the script is using this file.

Also recommend making the default ENV value production.

Reply:
we can change the default env value to `prod`. 

This will not overwrite values passed in via `docker run` or `docker-compose xxx`, both of those take precedence over default env vars set in the Dockerfile.